boards/omote/Makefile.include

```diff
@@ -0,0 +1,35 @@
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# define the default flash-tool
+PROGRAMMER ?= cc2538-bsl
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# For backward compatibility
+ifneq (,$(PORT_BSL))
+  $(warning Warning! PORT_BSL is deprecated use PROG_DEV)
+  PROG_DEV ?= $(PORT_BSL)
+endif
+
+ifeq ($(PROGRAMMER),cc2538-bsl)
+  PROG_BAUD ?= 115200
+  include $(RIOTMAKE)/tools/cc2538-bsl.inc.mk
+else ifeq ($(PROGRAMMER),jlink)
+  FLASHER = $(RIOTBOARD)/cc2538dk/dist/flash.sh
+  FFLAGS  = $(BINDIR) $(FLASHFILE)
+  DEBUGGER = $(RIOTBOARD)/cc2538dk/dist/debug.sh
+  DEBUGSERVER = JLinkGDBServer -device CC2538SF53
+  RESET ?= $(RIOTBOARD)/cc2538dk/dist/reset.sh
+  RESET_FLAGS ?= $(BINDIR)
+endif
+
+OFLAGS = --gap-fill 0xff
+FLASHFILE ?= $(BINFILE)
+DEBUGGER_FLAGS = $(BINDIR) $(ELFFILE)
+OBJDUMPFLAGS += --disassemble --source --disassembler-options=force-thumb
+
+# include common cc2538 includes
+INCLUDES += -I$(RIOTBOARD)/common/cc2538/include
```

Comment:
Actually you should be able to drop all that and replace it with

```Makefile
include $(RIOTBOARD)/common/remote/Makefile.include
```

see #14554